scripts/ecc-heap.sh

```diff
@@ -68,7 +68,10 @@ cat << EOF >$CONFIG_H
 #define MBEDTLS_ECP_DP_SECP521R1_ENABLED
 #define MBEDTLS_ECP_DP_CURVE25519_ENABLED
 
-//#define MBEDTLS_ECP_WINDOW_SIZE            6
+#define MBEDTLS_HAVE_ASM // just make things a bit faster
+#define MBEDTLS_ECP_NIST_OPTIM // faster and less allocations
```

Comment:
`MBEDTLS_ECP_NIST_OPTIM` is a performance vs code size compromise, isn't it? My understanding is that `ecc-heap.sh` is intended to benchmark for a resource-constrained device, so doesn't it make sense to leave `MBEDTLS_ECP_NIST_OPTIM` off?

Reply:
Have we measured the impact of `MBEDTLS_ECP_NIST_OPTIM`? ISTM that the modular reduction routines are fairly small. It even seems like an option one might want to enable unconditionally.

Reply:
Ideally it should benchmark both ways, but I didn't feel like making it even slower.

The size of the modular reduction routines is not insignificant for common NIST curves:

Reply:
Actually I was reminded by Hanno's measurements in another PR that Brainpool curves don't have "mod p" routines, so their figures are very similar to those of the NIST curves with `NIST_OPTIM` disabled. So I went with the following compromise: only build with `NIST_OPTIM` enabled, but include Brainpool curves. That gives essentially the same information, but with less builds.